### PR TITLE
minor carnet improvements

### PR DESCRIPTION
**Pull request by Nisha1306**

### CODE REVIEW

File: src/screens/Ticketing/Ticket/Carnet/CarnetDetails.tsx [hunk 1]
```diff
@@ -81,7 +81,7 @@ export function CarnetDetails(props: {
         )}
         <TicketInfo
           travelRights={travelRights}
-          status={fareContractValidityStatus}
+          status={usedAccessValidityStatus}
```
Comment (Nisha1306) [inline src/screens/Ticketing/Ticket/Carnet/CarnetDetails.tsx:84]: fareContractValidityStatus tells the overall status of the carnet ticket, while usedAccessValidityStatus is if the ticket is currently in use or not. Its value is 'valid' and 'inactive'. Hence passing that down to show zone/non-inspectable symbol and warning.

File: src/screens/Ticketing/Ticket/TicketInfo.tsx [hunk 2]
```diff
@@ -146,26 +146,22 @@ const TicketInfoHeader = ({
...
-
   return (
     <View style={styles.header}>
       <View style={styles.ticketHeader}>
         {productName && (
           <ThemeText
             type="body__primary--bold"
             style={styles.product}
-            accessibilityLabel={productName + screenReaderPause}
+            accessibilityLabel={productName + ', AtB' + screenReaderPause}
```
[the remark this answers is not in the document]
Comment (Nisha1306) [reply src/screens/Ticketing/Ticket/TicketInfo.tsx]: Fixed it. Does it look better? Not sure about NFK casing added 😅

[the remark this answers is not in the document]
Comment (Nisha1306) [reply src/screens/Ticketing/Ticket/TicketInfo.tsx]: or I think I should change it to NfK instead?

**Review by rosvik — APPROVED** [review]
👍

File: src/screens/Ticketing/Ticket/TicketInfo.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (rosvik) [reply src/screens/Ticketing/Ticket/TicketInfo.tsx]: NFK should be correct 👍

**Comment by gorandalum** [comment]
@Nisha1306 @rosvik I think showing the organization name should be solved a little more correctly, as it will probably at some time come back and bite us. We want to show the name of the organization that issued the ticket, and not the name of the organization which released the app. It is intended in the future that AtB tickets can be shown in other apps and vice versa.

And where does it come from that we should show "AtB" in the name of the carnet ticket? Kinda hard to track as there is no reference to an issue or discussion in this PR.

**Comment by Nisha1306** [comment]
> @Nisha1306 @rosvik I think showing the organization name should be solved a little more correctly, as it will probably at some time come back and bite us. We want to show the name of the organization that issued the ticket, and not the name of the organization which released the app. It is intended in the future that AtB tickets can be shown in other apps and vice versa.
> 
> And where does it come from that we should show "AtB" in the name of the carnet ticket? Kinda hard to track as there is no reference to an issue or discussion in this PR.

Ohh nice catch @gorandalum ! This is actually a part of Ticketing v2. https://github.com/AtB-AS/kundevendt/issues/330 Does this mean that we are right nor not ready to show the organisation name? Or we can still derive it somehow?

**Comment by gorandalum** [comment]
We can get it through the Entur api, but I say we just skip it for now. Showing tickets from other organisations is not right around the corner.

**Comment by Nisha1306** [comment]
> We can get it through the Entur api, but I say we just skip it for now. Showing tickets from other organisations is not right around the corner.

This has been removed as part of https://github.com/AtB-AS/mittatb-app/pull/2901

**Pull request merged**